Automattic/wp-calypso
# Calypso editor: do not set Calypsoify on Jetpack 9.2

#### Changes proposed in this Pull Request

Calypsoify is currently broken in Jetpack 9.2:
https://github.com/Automattic/jetpack/pull/17939

Until a point release is available, this may avoid creating Fatal errors on sites that do not have Calypsoify enabled yet.

#### Testing instructions

* Start from a Jetpack site (not Atomic) connected to WordPress.com and running Jetpack 9.2.
* Go to Jetpack > Settings and enable SSO
* Go to WordPress.com > Posts for that site
* Click on the button to start a new post.
* You should be redirected to the wp-admin editor without any Calypsoify styling.
* Now try the same thing with an Atomic site; the iFramed block editor should load instead of the redirection.
